### PR TITLE
add sequence diagrams for boot and runtime

### PR DESCRIPTION
**Pull request by thomas-fossati**
as per Eliot's email

### CODE REVIEW

File: art/psa-attester.ascii-art [hunk 1]
```diff
@@ -24,3 +24,6 @@
 | | Target Environment                                            | |
 | '---------------------------------------------------------------' |
 '-------------------------------------------------------------------'
+Legenda:
```
Comment (adrianlshaw) [inline art/psa-attester.ascii-art:27]: Is _**legenda**_ correct? Not legend? I'm not familiar with what RFCs use.

Comment (thomas-fossati) [inline art/psa-attester.ascii-art:27]:
```suggestion
Legend:
```

**Review by adrianlshaw — APPROVED** [review]

File: art/psa-runtime.ascii-art [hunk 2]
```diff
@@ -0,0 +1,11 @@
+      MBS           IAS      Verifier
+       | read        |          |
+       | measurement |          |
```
Comment (thomas-fossati) [inline art/psa-runtime.ascii-art:3]: This should either be plural (measurements) or be in a "loop i" that, for each TE, reads the relevant measurement.

File: art/psa-boot.ascii-art [hunk 3]
```diff
@@ -0,0 +1,13 @@
+       TE_i       MBL           MBS
+         |         |             |
+.--------|---------|-------------|----.
+| loop i |         |             |    |
+|        | measure |             |    |
+|        | & load  |             |    |
+|        |o--------+             |    |
+|        |         | write       |    |
+|        |         | measurement |    |
+|        |         +------------>|    |
```
Comment (hannestschofenig) [inline art/psa-boot.ascii-art:10]:
```suggestion
|        |         +W----------->|    |
```

Comment (hannestschofenig) [reply art/psa-boot.ascii-art:10]: Just for consistency I am wondering whether we should add this "W"

**Comment by hannestschofenig** [comment]
Good clarifications, Thomas.

File: art/psa-runtime.ascii-art (hunk 2 above)
Comment (thomas-fossati) [reply art/psa-runtime.ascii-art]: addressed in 232ae237e

File: art/psa-attester.ascii-art (hunk 1 above)
Comment (thomas-fossati) [reply art/psa-attester.ascii-art]: addressed in [2548f51](https://github.com/thomas-fossati/draft-psa-token/pull/125/commits/2548f51f98e1960ffea60992c14a68140e3281a2)

File: art/psa-boot.ascii-art (hunk 3 above)
Comment (thomas-fossati) [reply art/psa-boot.ascii-art]: In this case, there was no need to qualify the arrow as the description already mentions "write [...]". Similarly, in the other diagram, it says "read [...]".

**Review by SimonFrost-Arm — APPROVED** [review]
LGTM

**Review by mathias-arm — APPROVED** [review]

**Pull request merged**